src/plugin/index-more/src/java/org/apache/nutch/indexer/more/MoreIndexingFilter.java

```diff
@@ -316,6 +324,39 @@ public void setConf(Configuration conf) {
         LOG.error(org.apache.hadoop.util.StringUtils.stringifyException(e));
       }
     }
+    
+    URL dateStylesResource = conf.getResource("date-styles.txt");
+    if (dateStylesResource == null) {
+      LOG.warn("Can't find resource: date-styles.txt");
```

Comment:
Right now the config file is optional. If this is the desired behavior then I would also add to the message (since it is a warning) something that the default list of dates will be used.